src/itest/resources/features/delete_psc.feature

```diff
@@ -24,7 +24,9 @@ Feature: Delete PSC
   Scenario Outline: Delete PSC unsuccessfully - PSC resource does not exist
```

Comment:
Rewrite needed here, unsuccessfully -> successfully

Reply:
done, it is still unsuccessful in the delete but I've made the test name a bit clearer for now

Reply:
This is a successful scenario, given the 200 ok response.